server/services/communityApiService.ts

```diff
@@ -13,6 +13,13 @@ export interface DeliusUser {
   roles: Array<DeliusRole>
 }
 
+export interface DeliusOffender {
```

Comment:
Can we use the term `ServiceUser` here? I know it might be a bit confusing having the `DelisUser` and `DeliusServiceUser` but we're trying to move away from the term `offender`in our service.

Reply:
yeh sounds good